iocore/net/SSLSessionTicket.h

```diff
@@ -25,6 +25,7 @@
 
 #include <openssl/safestack.h>
 #include <openssl/tls1.h>
+#include <openssl/ssl.h>
```

Comment:
`SSL_CTX_set_tlsext_ticket_key_cb`, which is checked on line 31, is defined in "ssl.h". Without including the header file here, the check can fail on some files.